hangupsbot/plugins/_chatbridge/chatbridge_syncrooms.py

```diff
@@ -59,25 +59,55 @@ def _send_to_external_chat(self, config, event):
         message = event.passthru["original_request"]["message"]
         image_id = event.passthru["original_request"]["image_id"]
 
+        is_action = event.passthru["chatbridge"].get("source_action")
+
         if not message:
             message = ""
 
+        if any(a.type == 4 for a in event.conv_event._event.chat_message.annotation):
+            # This is a /me message sent from desktop Hangouts.
+            is_action = True
+            # The user's first name prefixes the message, so try to strip that.
```

Comment:
after further checking examining this, it seems impossible that an actual event user will not be matched to at least the first name; and very unlikely that there is a mismatch between a stored username and the first name that the server returns - rare enough that unmatched names deserve a `WARNING` instead of a `pass` - unless you've observed otherwise in your own testing.